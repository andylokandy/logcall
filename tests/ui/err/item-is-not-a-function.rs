#[logfn::logfn("info")]
struct S;

fn main() {}
