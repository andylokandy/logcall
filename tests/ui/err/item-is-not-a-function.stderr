error: expected `fn`
 --> tests/ui/err/item-is-not-a-function.rs:2:1
  |
2 | struct S;
  | ^^^^^^
